src/file-utils/edfi_lms_file_utils/file_reader.py

```diff
@@ -268,7 +268,21 @@ def read_section_associations_file(
     -------
     Pandas DataFrame with columns matching the model definition / CSV file.
     """
-    return _read_csv(full_path, nrows, log_level=logging.DEBUG)
+    data_types = {
+        "EnrollmentStatus": "string",
+        "LMSSectionSourceSystemIdentifier": "string",
+        "LMSUserSourceSystemIdentifier": "string",
+    }
+
+    extra_date_columns = ["StartDate", "EndDate"]
+
+    return _read_csv(
+        full_path,
+        nrows,
+        log_level=logging.DEBUG,
```

Comment:
Why hard-coding the log level to DEBUG?

Reply:
Good question, that's not a change of mine though. Just an auto reformat.

Reply:
I did it, the _read_csv has a default info level... Setting it to debug level was part of the acceptance criteria for the" do not reprocess files" ticket. The reason was to don't get a lot of messages in the log.

Reply:
I overlooked this in the prior review. The code should have used `logger.Debug(message)` instead of `logger.log(log_level, message)`. Its not a big deal, but perhaps we can opportunistically refactor that in the future.